astropy/modeling/functional_models.py

```diff
@@ -1123,6 +1167,18 @@ def __init__(self, amplitude=amplitude.default, x_0=x_0.default,
             amplitude=amplitude, x_0=x_0, y_0=y_0, r_in=r_in, width=width,
             **kwargs)
 
+    def bounding_box_default(self):
```

Comment:
Could there be a generic bounding box test, where you evaluate all models with bounding boxes on an image that mostly contains the image and bounding box, and then check that the sum is only slightly smaller or identical with the bounding box?